devel/smake/Portfile

```diff
@@ -1,41 +1,96 @@
-                    sha256  27566aa731a400c791cd95361cc755288b44ff659fa879933d4ea35d052259d4
+name                    smake
+version                 1.7-${codeberg.version}
+revision                0
+categories              devel
+platforms               darwin
+maintainers             {hotmail.com:amtor @RobK88} \
+                        openmaintainer
+license                 CDDL
+description             A highly portable make program with automake features
+long_description        Smake is a highly portable make program with automake \
+                        features. It is currently mainly targeted to be used \
+                        with the Schily SING makefile system.
 
-use_configure	no
+homepage                https://codeberg.org/schilytools/schilytools
+master_sites            ${homepage}/archive/
+checksums               rmd160  e154278ecbe7d778bc1d6766ed163c9963b1cc82 \
+                        sha256  a4270cdcca5dd69c0114079277b06e5efad260b0a099c9c09d31e16e99a23ff5 \
+                        size    5896292
 
-# https://trac.macports.org/ticket/31616
-compiler.blacklist-append llvm-gcc-4.2 {clang < 318}
-
-build.type		gnu
-build.args      CC=${configure.cc}
-use_parallel_build  no
-
-destroot.args       CC=${configure.cc}
-destroot.destdir	INS_BASE=${destroot}${prefix} \
-					MANDIR=man \
-					INSUSR=`id -u` INSGRP=`id -g`
-post-destroot	{
-	file delete -force ${destroot}${prefix}/share/man/man5 \
-		${destroot}${prefix}/lib ${destroot}${prefix}/include
+post-extract {
+                        copy ${filespath}/Gmake.smake ${worksrcpath}
 }
 
-livecheck.url       https://sourceforge.net/projects/s-make/files/
-livecheck.regex     ${name}-(\\d+(?:\\.\\d+)*)${extract.suffix}
+patchfiles              001-Makefile-Patch.diff
+
+post-patch {
+                        reinplace -locale C "s|/opt/schily|${prefix}|g" \
+                        ${worksrcpath}/DEFAULTS/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS/Defaults.mac-os10 \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.darwin \
+                        ${worksrcpath}/DEFAULTS_ENG/Defaults.mac-os10 \
+                        ${worksrcpath}/libfind/find.c \
+                        ${worksrcpath}/libfind/find_main.c \
+                        ${worksrcpath}/librscg/scsi-remote.c \
+                        ${worksrcpath}/patch/tests/random/cmptest.sh \
+                        ${worksrcpath}/patch/tests/random/gentest.sh \
+                        ${worksrcpath}/smake/job.c \
+                        ${worksrcpath}/smake/make.c \
+                        ${worksrcpath}/smake/smake.1 \
+                        ${worksrcpath}/TEMPLATES/Defaults.gcc \
+                        ${worksrcpath}/TEMPLATES/Defaults.clang \
+                        ${worksrcpath}/TEMPLATES/Defaults.xcc
+}
+
+conflicts_build-append  cdrtools
+
+# https://trac.macports.org/ticket/31616
+compiler.blacklist      llvm-gcc-4.2 macports-llvm-gcc-4.2 {clang < 300}
```

Comment:
Similar to `patchfiles-append`, `compiler.blacklist-append` should be used, rather than `compiler.blacklist`.

Reply:
Will do.